examples/get-bucket-lifecycle.mjs

```diff
@@ -25,9 +25,8 @@ const s3Client = new Minio.Client({
   secretKey: 'YOUR-SECRETACCESSKEY',
 })
 
-s3Client.removeBucketLifecycle('bucketname', function (err) {
-  if (err) {
-    return console.log(err)
-  }
-  console.log('Success')
-})
+try {
+  await s3Client.getBucketLifecycle('test-bucket')
```

Comment:
Would it make sense to have a `console.log` statement here to display the bucket lifecycle configuration that was just retrieved?

```suggestion
  const bucketLifecycleConfiguration = await s3Client.getBucketLifecycle('test-bucket')
  console.log(bucketLifecycleConfiguration)
```

Reply:
Yes. Thank you. i have updated as per suggestions.